test/expect/TestScript.test_index_select_shape_prop.expect

```diff
@@ -1,5 +1,5 @@
 graph(%x : Double(2, 2)
       %y : Long(4)) {
-  %2 : Double(2, 4) = aten::index_select[dim=1](%x, %y)
-  return (%2);
+  %3 : Double(2, 4) = aten::index_select[dim=1](%x, %y)
```

Comment:
Can we run `_jit_pass_canonicalize` on this one to avoid meaningless changes in the future?